gdk/common/GithubUtils.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+
+class GithubUtils:
+    def get_latest_release_name(self, owner, repository):
+        latest_release_api_url = f"https://api.github.com/repos/{owner}/{repository}/releases/latest"
```

Comment:
This one should work even when the customer doesn't have git installed or signed into their account, correct?

Reply:
Yup, I'm not adding any authentication details to the request. It should work up until the rate limit, at which point the backup hardcoded value will have to be used.